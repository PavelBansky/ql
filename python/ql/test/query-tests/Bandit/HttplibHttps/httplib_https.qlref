Bandit/httplib_https.ql