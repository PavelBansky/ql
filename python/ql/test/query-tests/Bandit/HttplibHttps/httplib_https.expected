| Test.py:2:5:2:27 | Attribute | Use of HTTPSConnection on older versions of Python prior to 2.7.9 and 3.4.3 do not provide security |
| Test.py:5:5:5:31 | Attribute | Use of HTTPSConnection on older versions of Python prior to 2.7.9 and 3.4.3 do not provide security |
| Test.py:8:1:8:37 | Attribute | Use of HTTPSConnection on older versions of Python prior to 2.7.9 and 3.4.3 do not provide security |
